build/scripts/GuardingV2ExtensionsHelper.psm1

```diff
@@ -205,4 +224,31 @@ function Update-AppSourceCopVersion
     return $appSourceCopJsonPath
 }
 
+function Get-StrictModeBranchStatus()
```

Comment:
I know where this comes for but consider giving it a better name that conveys "is strict mode enabled for branch".